plugin.py

```diff
@@ -55,6 +55,7 @@ class RestRequestCommand(sublime_plugin.WindowCommand):
     def __init__(self, *args: Tuple[Any], **kwargs: Dict[Any, Any]) -> None:
         super().__init__(*args, **kwargs)
         self._tick = 0
+        self.view = None
```

Comment:
Nit: could you rename this to `self._view`? Seems it should be 'private' and only accesible via `get_view`.

Reply:
Oops, thanks for the catch! updated the PR